tests/unit/checkboxradio/checkboxradio_core.js

```diff
@@ -0,0 +1,196 @@
+/*
+ * checkboxradio_core.js
+ */
+
+
+(function($) {
+
+module("Checkboxradio: core");
+test( "Checkbox - Initial class structure", function( assert ) {
+	expect( 4 );
+	var input = $("#check"),
+		label = $("label[for=check]");
+	ok( input.is( ":visible" ) );
+	assert.lacksClasses( label, "ui-button" );
+	input.checkboxradio();
+	assert.hasClasses( input, "ui-helper-hidden-accessible ui-checkboxradio" );
+	assert.hasClasses( label, "ui-button ui-widget ui-checkboxradio-label ui-corner-all",
+		true, "Label has proper classes" );
+});
+
+test( "Radios - Initial class structure", function( assert ) {
+	expect( 10 );
+	var inputs = $( "#radio0 input" ),
+		labels = $( "#radio0 label" );
+	ok( inputs.is( ":visible" ) );
+	labels.each( function() {
```

Comment:
As above, does this check actually accomplish anything? We generally assume a clean state when tests start.